### PR TITLE
feat: [CDS-48476]: show last step of config file wizard when it is being edited

### PR DESCRIPTION
**Pull request by chetanvekariya-harness**
### Summary
- Pass prevStepData manually to the last step while editing config file, so that we have connector value for dependent fields in last step
- Harness store does not need connector value for last step
- Make this change for both config file sections - In Service and Environment Overrides

#### Screenshots

https://user-images.githubusercontent.com/81295223/235070235-9dc2442d-a8d2-4954-8310-17167760f992.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by chetanvekariya-harness** [comment]
retrigger jest

**Comment by chetanvekariya-harness** [comment]
retrigger coverage

**Comment by chetanvekariya-harness** [comment]
retrigger jest

**Comment by chetanvekariya-harness** [comment]
retrigger jest

File: src/modules/70-pipeline/components/ConfigFilesSelection/ConfigFilesInterface.tsx [hunk 1]
```diff
@@ -55,17 +56,36 @@ export interface ConfigInitStepData {
   store: ConfigFileType
   files: ConfigFileFileStoreNode[] | string[] | string
   fileType: string
+  connectorRef?: string
+}
+
+export interface ConfigFileConnectorRefType {
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/components/ConfigFilesSelection/ConfigFilesInterface.tsx:62]: Similar interface is also added in `src/modules/70-pipeline/components/ManifestSelection/ManifestInterface.ts`. We we common out them?

**Review by litvinovstas — APPROVED** [review]

**Pull request merged**